hub/static/js/explore.esm.js

```diff
@@ -1,6 +1,16 @@
-import { Modal } from '../bootstrap/bootstrap.esm.min.js'
-import { createApp } from '../vue/vue.esm-browser.prod.js'
-import L from '../leaflet/leaflet-1.9.3.esm.js'
+//import { Modal } from '../../../vendor/bootstrap/js/bootstrap.esm.min.js'
+import { Modal } from 'bootstrap'
+//import { Modal } from 'bootstrap/js/dist/modal'
+import "bootstrap/js/dist/collapse"
+import "bootstrap/js/dist/tab"
```

Comment:
If you're importing all of bootstrap in your Modal import, as currently, don't also import the modules here, or they'll be imported twice, sadly.